docs/develop/create-sso-office-add-ins-nodejs.md

```diff
@@ -128,449 +129,454 @@ This article walks you through the process of enabling single sign-on (SSO) in a
-    * The code minimizes the data that must come from Microsoft Graph by specifying only the property we need ("name") and only the top 10 folder or file names.
-
-    ```javascript
-    const graphToken = req.get('access_token');
-    const graphData = await getGraphData(graphToken, "/me/drive/root/children", "?$select=name&$top=10");
-    ```
-
-1. Replace `TODO 16` with the following. About this code, note:
-
-    * If Microsoft Graph returns an error, such as invalid or expired token, there will be a code property in the returned object set to a HTTP status (e.g., 401). The code relays the error to the client. It will be caught in the `.fail` callback of `makeGraphApiCall`.
-    * Microsoft Graph data includes OData metadata and eTags that the add-in does not need, so the code constructs a new array containing only the file names to send to the client.
-
-    ```javascript
-    if (graphData.code) {
-        next(createError(graphData.code, "Microsoft Graph error: " + JSON.stringify(graphData)));
-    }
-    else {
-        const itemNames = [];
-        const oneDriveItems = graphData['value'];
-        for (let item of oneDriveItems) {
-            itemNames.push(item['name']);
-        }
-
-        res.send(itemNames)
-    }
-    ```
-
-1. Save and close the file.
+   > As the name suggests, the ssoAuthES6.js uses JavaScript ES6 syntax because using `async` and `await` best shows the essential simplicity of the SSO API. When the localhost server is started, this file is transpiled to ES5 syntax so that the sample will support Internet Explorer 11.
+
+A key part of the sample code is the client request. The client request is an object that tracks information about the request for calling REST APIs on the middle-tier server. It's necessary because the client request state needs to be tracked or updated through the following scenarios:
+
+- SSO retries where the REST API call fails because it needs additional consent. The sample code calls `getAccessToken` with updated authentication options, gets the required user consent, and then calls the REST API again. The goal is to not fail in scenarios where a REST API needs additional consent.
+- SSO fails and fallback authentication is required. The access token is acquired through MSAL in a pop-up dialog box. The goal is to not fail in this scenario and gracefully fall back to the alternative authentication approach.
+
+The client request object tracks the following data:
+
+- `authOptions` - [Auth configuration parameters](/javascript/api/office/office.authoptions) for SSO.
+- `authSSO` - true if using SSO, otherwise false.
+- `accessToken` - The access token to the middle-tier server. The method to obtain this token is different for SSO than fallback auth.
```

Comment:
```suggestion
- `accessToken` - The access token to the middle-tier server
```
This is repeated below the code sample so it's redundant here. Also, msft voice + style says to skip end punctuation  on lists so this is more consistent with the rest of the list.